ChangeLog.txt

```diff
@@ -1,13 +1,14 @@
-v2.x - 2017-xx-xx
+v1.3.x - 2017-xx-xx
```

Comment:
It can be updated once that decision has been made. This way it's just different from master/fixes